src/Options.ts

```diff
@@ -35,12 +35,12 @@ const options = {
   },
 };
 
-export type Options = ReturnType<typeof parse>;
+export type Options = ReturnType<typeof parseOptions>;
 
 /**
  * Parse and normalize command line arguments passed to the script.
  */
-function parse() {
+export function parseOptions() {
```

Comment:
Minor cleanup, not really related to formatting 😐 

Instead of allowing any function to import options, we parse once at the top and pass down as needed. Theoretically this makes testing easier (although I haven't needed this in any tests, yet).